group/templates/group/groupDetail.html

```diff
@@ -117,20 +118,22 @@ <h4 class="panel-title">
               <thead>
                 <tr>
                   <th class='col-md-2'>Title</th>
-                  <th class='col-md-4'>Time</th>
+                  <th class='col-md-4'>Start Time</th>
+                  <th class='col-md-4'>End Time</th>
```

Comment:
By the [Bootstrap 3 grid system](http://getbootstrap.com/css/#grid), I think that:
- .col-* should be placed in the .row
- .row should be placed in the .container or .container-fluid
- If the sum of col is less than twelve, you should wrap them into .row.

Please refer to examples in the above link.